renku/core/models/workflow/converters/cwl.py

```diff
@@ -18,15 +18,23 @@
 """Converter for workflows to cwl."""
 
 import os
+import re
 import tempfile
 from collections import defaultdict
 from pathlib import Path
+from typing import Optional, Union
 from uuid import uuid4
 
 import cwlgen
 
+from renku.core import errors
+from renku.core.management.workflow.concrete_execution_graph import ExecutionGraph
```

Comment:
We should move this file, maybe to somewhere in management, since it's not really a domain model but more business logic.

Also so a model doesn't have a dependency on a management class (dependencies should go the other way)